src/main/js/SinglePullRequest.tsx

```diff
@@ -22,18 +22,20 @@
  * SOFTWARE.
  */
 import React from "react";
-import { ErrorNotification, Loading } from "@scm-manager/ui-components";
+import { ErrorNotification, Loading, Notification } from "@scm-manager/ui-components";
 import { Route, RouteComponentProps, Switch, withRouter } from "react-router-dom";
 import PullRequestDetails from "./PullRequestDetails";
 import { Link, Repository } from "@scm-manager/ui-types";
 import { PullRequest } from "./types/PullRequest";
 import { getPullRequest, getReviewer } from "./pullRequest";
 import Edit from "./Edit";
+import { withTranslation, WithTranslation } from "react-i18next";
 
-type Props = RouteComponentProps & {
-  repository: Repository;
-  userAutocompleteLink: string;
-};
+type Props = RouteComponentProps &
+  WithTranslation & {
+    repository: Repository;
+    userAutocompleteLink: string;
+  };
```

Comment:
You migrated one component to FC but not the others? I don't get it.

Reply:
Well, good point 😵